src/mmeGrpcProtos/mmeGrpc.proto

```diff
@@ -114,7 +114,11 @@ message ProcedureStatsRespBuf
 	uint32 num_of_ho_request_ack_received = 55;
 	uint32 num_of_ho_command_to_src_enb_sent = 56;
 	uint32 num_of_ho_notify_received = 57;
-	uint32 num_of_ho_complete = 58;	
+	uint32 num_of_ho_complete = 58;
```

Comment:
wherever possible we should use uint32 or similar variable names...this really makes code portable. Other places counters are using int variable type which is not so good idea.